src/BenchmarksClient/Startup.cs

```diff
@@ -167,8 +148,7 @@ public static int Main(string[] args)
                     {
                         var now = DateTime.UtcNow;
 
-                        // Clean the job in case the driver is not running
-                        if (now - job.LastDriverCommunicationUtc > TimeSpan.FromSeconds(30))
+                        if (now - job.LastDriverCommunicationUtc > TimeSpan.FromSeconds(job.MaxDuration))
```

Comment:
Revert to 30, this actually has nothing to do with `MaxDuration`. Later I'll update it to a much smaller value as the driver is actually communicating every second.